pkg/action/cl001/ac000/plan.go

```diff
@@ -16,17 +16,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac002",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
 		Comment: "check cluster access",
 	},
 	{
 		Action:  "ac003",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(30*time.Minute, 1*time.Minute),
```

Comment:
That is way too much IMO. Shouldn't take more than 5 minutes for all masters to be ready in case the first apiserver started to serve the tenant API.

Reply:
Make sense I reverted my change.